crest/Assets/Crest/Crest/Shaders/OceanInputs/Resources/TerrainSignedDistanceField.shader

```diff
@@ -0,0 +1,58 @@
+// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+// Renders ocean depth - signed distance from sea level to sea floor
+Shader "Crest/Inputs/Depth/Initialise Signed Distance Field From Geometry"
+{
+	SubShader
+	{
+		Pass
+		{
+			// TODO(TRC):Now figure-out how to handle this blend operation properly!
+			// BlendOp Min
```

Comment:
Just realised I need to fix this! Will do it soon.

Reply:
Current plan to do this is to have one shader for rendering the depths that blends using this - and then having an init shader handle this.